apps/designer/app/designer/features/sidebar-left/panels/asset-manager/asset-info.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  Box,
+  Flex,
+  Grid,
+  IconButton,
+  Text,
+  Separator,
+  Button,
+} from "@webstudio-is/design-system";
+import type { Asset } from "@webstudio-is/prisma-client";
+import {
+  getAspectRatio,
+  getHumanReadableFileSize,
+  getStartAndEndOfString,
+} from "./utils";
+import {
+  CloudIcon,
+  AspectRatioIcon,
+  Cross1Icon,
+  SizeIcon,
+  TrashIcon,
+} from "@webstudio-is/icons";
+import { useSubmit } from "@remix-run/react";
+
+type AssetInfoProps = Asset & {
+  closeInfo: () => void;
+  setIsDeleting: (isDeleting: boolean) => void;
+};
+
+export const AssetInfo = ({
+  size,
+  width,
+  height,
+  id,
+  name,
+  closeInfo,
+  setIsDeleting,
+}: AssetInfoProps) => {
+  const submit = useSubmit();
+  const deleteAsset = () => {
+    const formData = new FormData();
+    formData.append("assetId", id);
+    formData.append("assetName", name);
+    closeInfo();
+    setIsDeleting(true);
+    submit(formData, { method: "delete" });
+  };
+
+  return (
+    <>
+      <Flex
+        css={{ height: 40, paddingLeft: "$3" }}
+        align="center"
+        justify="between"
+      >
+        <Text size="1" weight="700">
+          Asset Details
+        </Text>
+
+        <IconButton
+          onClick={closeInfo}
```

Comment:
standard convention is handleSomething, e.g. handleClose in this case, but since this one is just calling the future onClose we would be just passing onClose if not one problem: onClose has semantically no knowledge of the click event, so it should not receive the event object, otherwise its a leaking abstraction, so the best thing in such situations is to have this
```suggestion
          onClick={() => onClose()}
```